### PR TITLE
utc capi: Added shape fill utc.

### PR DESCRIPTION
**Pull request by mihashco**

### CODE REVIEW

File: test/capi/testFill.cpp [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <stdint.h>
+#include <thorvg_capi.h>
+#include "catch.hpp"
+
+#define WIDTH  100
```
[the remark this answers is not in the document]
Comment (hermet) [reply test/capi/testFill.cpp:27]: please do not use macro in test unless it helps the reduce  LOC.

**Review by hermet — CHANGES_REQUESTED** [review]
Please check comments.

File: test/capi/testFill.cpp [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <stdint.h>
```
[the remark this answers is not in the document]
Comment (mihashco) [reply test/capi/testFill.cpp:23]: yes it is for uint8_t

File: test/capi/testFill.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (mihashco) [reply test/capi/testFill.cpp:27]: removed

File: test/capi/testFill.cpp [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+#include <thorvg_capi.h>
+#include "catch.hpp"
+
+#define TEST_R 120
+#define TEST_G 154
+#define TEST_B 180
+#define TEST_A 100
+
+TEST_CASE("CAPI: set_get_fill_color_valid", "[capi-fill]")
```
Comment (JSUYA) [inline test/capi/testFill.cpp:31]: Hi, "CAPI: " is unnecessary.

Comment (hermet) [inline test/capi/testFill.cpp:31]: please keep the consistency for the naming of TEST_CASE()

**Review by hermet — CHANGES_REQUESTED** [review]
please keep the consistency for the naming of TEST_CASE()

**Comment by hermet** [comment]
each of your patches have different naming of TEST_CASE(), please keep its consistency.

File: test/capi/testFill.cpp [hunk 4]
```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020-2021 Samsung Electronics Co., Ltd. All rights reserved.
```
Comment (hermet) [inline test/capi/testFill.cpp:2]: 2021

**Review by hermet — APPROVED** [review]

**Pull request merged**